.vscode/launch.json

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
I don't think this file is actually necessary. By default, when you open a VSCode workspace, [the launch configurations from each of the subprojects is listed](https://code.visualstudio.com/docs/editor/multi-root-workspaces#_debugging) in the Debug pane.

If its not showing up, it's likely because you're simply opening the folder in VSCode, rather than opening the workspace. This is a pretty small detail, but if you're using the CLI, and your in the root of this repo, you'd run `code node-slack-sdk.code-workspace` instead of `code .`. Even if you do the latter, or you open VSCode and use the GUI to open the project, VSCode will warn you using a toast in the bottom right corner, and make it one click to open the workspace.

<img width="475" alt="Screen Shot 2019-11-22 at 1 47 22 PM" src="https://user-images.githubusercontent.com/494795/69462740-a9675680-0d2e-11ea-9b75-043ca905e9f4.png">

The problem with adding this file redundantly, just in case you opened the folder instead of the workspace, is that the Debug pane lists a duplicated list of tasks:

<img width="402" alt="Screen Shot 2019-11-22 at 1 43 32 PM" src="https://user-images.githubusercontent.com/494795/69462796-d156ba00-0d2e-11ea-90e8-97263869747d.png">

Reply:
Ahhh. I almost always do `code .` vs opening a workspace. Nice tip. I was already thinking that top level `launch.json` was a little redundant. Seeing the dupes in the workspace method reinforces that. Thanks!